### PR TITLE
Removes double spaces in defines and removes some unused ones

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

1. Removes unused defines (DIAG_CIRCUIT_HUD, LIGHTING_BASE_MATRIX, pda_styles and the defines it uses, MAFIA_MEDAL_CHARISMATIC & MAFIA_MEDAL_VIP).

2. Removes double spaces (or tab->space & space->tab) from everywhere I saw them in the defines folder (unless used by autodoc)

3. Removes the unecessary amount of tabs from lists in HubMigrator, this PR was at first gonna be around achievements but this just bugged me, which spiraled into this.

## Why It's Good For The Game

me when i spread my code standards onto the codebase i contribute to

this doesn't affect the game however it does make the defines look cleaner and brings it up to date with our most recent standards.

## Changelog

not player-facing.

### CODE REVIEW

File: code/__DEFINES/lighting.dm [hunk 1]
```diff
@@ -32,17 +32,6 @@
 /// Set to zero to disable soft lighting. Luminosity changes then work if it's lit at all.
 #define LIGHTING_SOFT_THRESHOLD 0
 
-/// If I were you I'd leave this alone.
-#define LIGHTING_BASE_MATRIX \
-	list                     \
```
Comment (MrStonedOne) [inline code/__DEFINES/lighting.dm:37]: :eyes:

Comment (LemonInTheDark) [reply code/__DEFINES/lighting.dm:37]: tfw <https://github.com/tgstation/tgstation/pull/58991>

Comment (Kylerace) [reply code/__DEFINES/lighting.dm:37]: the instructions were VERY clear

Comment (JohnFulpWillard) [reply code/__DEFINES/lighting.dm:37]: They were not me

**Comment by ZephyrTFA** [comment]
> Removes double spaces (or tab->space & space->tab) from everywhere I saw them in the defines folder (unless used by autodoc)

Do you want me to add this to check_grep?

**Comment by JohnFulpWillard** [comment]
probably would be good, there's probably a lot more examples of this, I just did defines.

**Pull request merged**